### PR TITLE
Increased timeout to match timeout in config file

### PR DESCRIPTION
**Pull request by vkedia**

### CODE REVIEW

File: packages/spanner/src/transaction.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ function Transaction(session, options) {
   this.retries_ = 0;
   this.runFn_ = null;
 
-  this.timeout_ = 60000;
+  this.timeout_ = 3600000;
```
Comment (callmehiphop) [inline packages/spanner/src/transaction.js]: Could we just require the config file so we don't have to mange it in multiple places?

Comment (stephenplusplus) [reply packages/spanner/src/transaction.js]: That makes sense to me.

Comment (vkedia) [reply packages/spanner/src/transaction.js]: I agree. Currently if config file is present does this value get overridden by the config file value somewhere?

**Comment by stephenplusplus** [comment]
@callmehiphop do we still need this?

**Comment by callmehiphop** [comment]
@stephenplusplus yep!

**Comment by stephenplusplus** [comment]
Could you create an issue / send the PR on the new repo?

**Comment by stephenplusplus** [comment]
Survived by https://github.com/googleapis/nodejs-spanner/pull/75

**Pull request closed without merging**